editor.html

```diff
@@ -347,8 +357,8 @@ <h2 id="flashing-text">Flashing micro:bit</h2>
         </div>
         <div id="modal-msg-overlay-container" class="modal-overlay-container modal-msg-overlay-container">
           <div id="modal-msg-overlay" class="modal-overlay">
-            <a class="vex-close modal-close" onclick="$('#modal-msg-overlay-container').hide()"></a>
-            <div id="modal-msg-text">
```

Comment:
Any reason this was removed? There is some CSS associated with that.

Reply:
```suggestion
             <a class="vex-close modal-close" onclick="$('#modal-msg-overlay-container').hide()"></a> 
            <div id="modal-msg-text">
```